### DISCUSSION TITLE
Advanced notification configuration that only failed backups create a notification

### DISCUSSION THREAD

**Discussion by schellmann** [nextcloud/all-in-one#2180] [Ideas] [answered]
It would be great if you could set it to only create a notification when a backup fails.

**Comment by szaimen** [comment]
Hi, see https://github.com/nextcloud/all-in-one/discussions/484

**Comment by szaimen** [comment, marked as answer]
Will be possible with https://github.com/nextcloud/all-in-one/pull/3516

**Reply by szaimen** [reply]
This is now released with v7.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
